src/main/java/com/hr/controller/FireController.java

```diff
@@ -23,14 +23,14 @@ public String deleteEmployee(@ModelAttribute(value="employee") Employee employee
 
         Employee user = employeeModifier.findEmployeeById(employee.getId());
 
-        if (!Objects.equals(user, Optional.empty())){
+        if (user != null){
             employeeModifier.findEmployeeById(user.getId());
             model.addAttribute("employee", employee);
-            return "FirePage";
+            return "hirepage";
```

Comment:
Why are you changing this?

Reply:
I change this since the name of html file is "hirepage", so the name mush match the name of html